js/components/harvest/schedule-modal.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+<div>
+    <modal v-ref:modal :title="_('Scheduling')" class="schedule-modal" :class="modalClass">
+        <div class="modal-body" v-if="!unscheduling">
+            <horizontal-form class="schedule-form" :fields="fields" :defs="defs" v-ref:form
+                :model="source">
+            </horizontal-form>
+        </div>
+        <div class="modal-body" v-if="unscheduling">
+            <p class="lead text-center">
+                {{ _('You are about to unschedule this harvest source') }}
+            </p>
+            <p class="lead text-center">
+                {{ _('Are you sure?') }}
+            </p>
+        </div>
+        <footer class="modal-footer text-center">
+            <button type="button" v-if="!unscheduling" class="btn btn-outline btn-flat pull-left" @click="schedule">
+                {{ _('Schedule') }}
+            </button>
+            <button type="button" v-if="unscheduling" class="btn btn-warning btn-flat pull-left" @click="unschedule">
+                {{ _('Confirm') }}
+            </button>
+            <button type="button"  v-if="!unscheduling && source.schedule"
+                class="btn btn-warning btn-flat pull-left" @click="unscheduling = true">
+                {{ _('Unschedule') }}
+            </button>
+            <button type="button" class="btn btn-sm btn-flat" :class="cancelClass" @click="cancel">
+                {{ _('Cancel') }}
+            </button>
+        </footer>
+    </modal>
+</div>
+</template>
+<script>
+import API from 'api';
+
+import HorizontalForm from 'components/form/horizontal-form.vue';
+
+import HarvestSource from 'models/harvest/source';
+
+import Modal from 'components/modal.vue'
+
+const MASK = ['id', 'name', 'schedule'];
+const DEFAULT_SCHEDULE = '0 0 * * *';
```

Comment:
Should not be fetched from the backend? Cf your first PR.

Reply:
Yes, But i think I'll do that in another PR (needs an entrypoint or some configuration and this is only available to admin user)